ajax/libs/multiple-select/package.json

```diff
@@ -28,7 +28,7 @@
       {
         "basePath": "",
         "files": [
-          "multiple.select.*"
+          "multiple-select*"
```

Comment:
Let's use `"multiple-select*".+(js|css|map)`.
We don't need the file `multiple-select.jquery.json`